src/chart/heatmap/HeatmapSeries.ts

```diff
@@ -41,14 +41,18 @@ import type Calendar from '../../coord/calendar/Calendar';
 
 type HeatmapDataValue = OptionDataValue[];
 
+export type BorderRadius = number | number[];
```

Comment:
This definition for the `BorderRadius` type is unnecessary and can be merged into line 49 below.

Reply:
@plainheart I've fixed it, review it in your free time plz.